docs/docker-image-details.md

```diff
@@ -14,11 +14,15 @@ You can deploy the service with the Gravitino Docker image.
 Container startup commands
 
 ```shell
-docker run --rm -d -p 8090:8090 -p 9001:9001 apache/gravitino:0.6.1-incubating
+docker run --rm -d -p 8090:8090 -p 9001:9001 apache/gravitino:0.7.0-incubating
 ```
 
 Changelog
 
+- apache/gravitino:0.7.0-incubating
+  - Based on Gravitino 0.7.0-incubating, you can know more information from 0.7.0-incubating release notes.
```

Comment:
is this line necessary?

Reply:
could you provide import features of 0.7.0-incubating?

Reply:
> is this line necessary?

This content is similar to version 0.6.0, is it necessary to add the details here in this docker document here?

Reply:
done